tests/subsys/logging/log_switch_format/prj.conf

```diff
@@ -9,6 +9,9 @@ CONFIG_LOG_MIPI_SYST_ENABLE=y
 # use the correct logging backend.
 CONFIG_LOG_BACKEND_MOCK=y
 CONFIG_LOG_BACKEND_UART=n
+CONFIG_LOG_BACKEND_RTT=n
+CONFIG_LOG_BACKEND_NATIVE_POSIX=n
+CONFIG_LOG_BACKEND_XTENSA_SIM=n
```

Comment:
why aren't these mutually exclusive (i.e. part of a choice)?

Reply:
This testcase is supposed to only use mock backend for testing purposes.

Reply:
yes, in this particular test i want to disable any backend that could interfere with the test (which expects to have only mock backend). As for mutually exclusiveness, some combinations are no possible but some are (e.g. rtt + uart exists).